include/RAJA/policy/openmp/params/forall.hpp

```diff
@@ -287,46 +227,21 @@ RAJA_INLINE void forall_impl(const ::RAJA::policy::omp::Guided<ChunkSize>& p,
 
     using RAJA::internal::thread_privatize;
     auto body = thread_privatize(loop_body);
-
-#pragma omp for schedule(guided) reduction(combine : f_params)
-    for (decltype(distance_it) i = 0; i < distance_it; ++i)
+    if constexpr (ChunkSize <= 0)
```

Comment:
Can you pick a consistent ordering? In some forall implementations, you do the ChunkSize > 0 case first and in others the ChunkSize <= 0 first.